src/components/ui/TimePicker/TimePicker.tsx

```diff
@@ -0,0 +1,41 @@
+import ReactDatePicker from "react-datepicker";
+import { forwardRef } from "react";
+import "react-datepicker/dist/react-datepicker.css";
+import DatePicker from "react-datepicker";
+import InputWithIcon from "@/components/ui/InputWithIcon";
+import { cn } from "@/utils/helper";
+import { IconPosition } from "@/models/enums/ButtonVariant";
+import { twMerge } from "tailwind-merge";
+import { IDatePickerProps } from "@/components/ui/DatePicker/Datepicker.d";
+
+const TimePicker = forwardRef<ReactDatePicker, IDatePickerProps>(
+  (props, ref) => {
+    return (
+      <DatePicker
+        customInput={
+          props.icon ? (
+            <InputWithIcon
+              icon={props.icon}
+              className={cn("input", { "input-has-error": props.hasError })}
+              iconPosition={IconPosition.LEFT}
+            />
+          ) : (
+            <input />
```

Comment:
Input should have more definition

Reply:
I have added id, name and type props to the input component